src/Components/NavBar/NavItem.tsx

```diff
@@ -94,6 +104,8 @@ export const NavItem: React.FC<NavItemProps> = ({
         </Sans>
       </Link>
 
+      {hover && onHover ? trackHover() : null}
```

Comment:
This can [be an effect](https://reactjs.org/docs/hooks-effect.html), and we can call the tracking function when `hover` is true.

```tsx
useEffect(() => {
  if (hover) {
    trackHover()
  }
}, [hover])
```

Reply:
Another thought is functions shouldn't be called like this inside of JSX, unless the function returns more JSX. Why call in here instead of around line 72?